server/lib/api_config.js

```diff
@@ -0,0 +1,5 @@
+exports.wordAssocAPIk = 
```

Comment:
You'll need to add the api_config.js with the API keys to .gitignore. Don't want to upload API keys to public repo (we'll probably need to regen these keys)